adapters/Laravel/Identity/src/Adapters/IdentityOperationsAdapter.php

```
@@ -0,0 +1,443 @@
+            return $this->mfaVerification->verifyTotp($userId, $code);
+        }
+
+        $this->logger->warning('Unsupported MFA method verification attempt', [
+            'user_id' => $userId,
+            'method' => $method->value
+        ]);
+
+        return false;
+    }
+
+    public function verifyBackupCode(string $userId, string $code, ?string $tenantId = null): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$tenantId'. (undefined)

(UnusedFormalParameter)